lhotse/features/kaldi/layers.py

```diff
@@ -13,10 +13,17 @@
 
 This implementation works well with autograd and batching, and can be used neural network
 layers.
+
+Update January 2022:
+These modules now expose a new API function called "online_inference" that
+may be used to compute the features when the audio is streaming.
+The implementation is stateless, and passes the waveform remainders
+back to the user to feed them to the modules once new data becomes available.
+The implementation is compatible with JIT scripting via TorchScript.
```

Comment:
Are there tests to ensure this?

Reply:
Yeah, this PR #521 added torchscript compatibility and tests for it, and in this PR the new methods are annotated with `torch.jit.export`, so they're automatically tested by the previously existing tests (I also manually torchscripted it and ran some forward passes to make sure).